src/Microsoft.Sbom.Api/Utils/ComponentDetectionCliArgumentBuilder.cs

```diff
@@ -85,7 +85,7 @@ public ComponentDetectionCliArgumentBuilder Scan()
 
         public ComponentDetectionCliArgumentBuilder AddDetectorArg(string name, string value)
         {
-            detectorArgs.TryAdd(name, value);
+            detectorArgs[name] = value;
```

Comment:
>detectorArgs [](http://example.com/codeflow?start=12&length=12)

Do want to revert this recent change?

Reply:
No, it's not necessary. Last PR fixed actual issue, but this change is just an "improvement" to make things more logical and matching my expectation as a consumer of this class :)